.github/workflows/ci.yml

```
@@ -0,0 +1,191 @@
+name: CI/CD Pipeline
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)